index.html

```diff
@@ -1,12 +1,76 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на интенсивном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">Базовый HTML и CSS</a>».</p>
-
-  </body>
+<html lang="ru" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Девайс</title>
+</head>
+<body>
+<header class="header">
+  <div class="header1">
+    <ul>
+      <li><button class="search">poisk po saitu</button></li>
+      <li><button class="enter">voiti</button></li>
+      <li><button class="compare">sravnit</button></li>
+      <li><button class="basket">basket</button></li>
+    </ul>
+  </div>
+  <div class="header2">
+    <ul><!--задать один стиль всем ссылкам, где ставить class-->
+      <li> <a href="">catalog</a></li>
+      <li> <a href="">dostavka</a></li>
+      <li> <a href="">garantiya</a></li>
+      <li> <a href="">kontakti</a></li>
+    </ul>
+  </div>
+</header>
+<div class="gallery">
+  <!-- не знаю как сделать галерею с слайдами,
+  понимаю, что слева картинка, справа текст
+  и есть 3 кнопки, -->
+</div>
+<div class="assortiment">
+  <ul>
+    <li><a href=""></a> <img src="virual ochki">virtualnaya realnost</li>
+    <li><a href=""></a> <img src="monopod selfi">monopodi dlya selfi</li>
+    <li><a href=""></a> <img src="kamera">ekshn-kameri</li>
+    <li><a href=""></a> <img src="braslet">itnes brasleti</li>
+    <li><a href=""></a> <img src="smart watch">umnie chasi</li>
+    <li><a href=""></a> <img src="kvadrik">kvadrokopteri</li>
+  </ul>
+</div>
+<div class="usloviya">
+  <button class="dostavkabt">dostavaka</button>
+  <button class="garantiyabt">garantiya</button>
+  <button class="kreditbt">kredit</button>
+  <!-- блок о доставке, не знаю как подключить кнопки перелистывсть-->
+<div class="dostavka">
+  <h1>dostavka</h1>
+  <p> mi s udovolstviem....</p>
+  <img src="">
+</div>
+<div class="garantiya">
+     <h1>garantiya</h1>
+<p> midostavim vashe govno</p>
+<img src="">
+</div>
+<div class="kredit">
+  <h1>kredit</h1>
+  <p>u nas est' kredit</p>
+  <img src="">
+</div>
+</div>
+<div class="logo sponsor">
+  <img src="">
```

Comment:
Please add fake images like it said in the task. See https://up.htmlacademy.ru/htmlcss/17/tasks/16